ios/FluentUI/Vnext/List/List.swift

```diff
@@ -97,41 +159,102 @@ public struct MSFListView: View {
     }
 }
 
-/// UIKit wrapper that exposes the SwiftUI List implementation
-@objc open class MSFList: NSObject, FluentUIWindowProvider {
+/// Properties that make up section content
+class MSFListSectionStateImpl: NSObject, ObservableObject, Identifiable, MSFListSectionState {
+    func createCell() -> MSFListCellState {
```

Comment:
I agree with @mischreiber on the consistency.
My vote for the "create" term is because "insert" or "append" don't usually imply the creation of an object.
So I believe we can make explicit in the public documentation that this method creates a new cell and appends it to the section's collection of cells.

Reply:
I think the explanation is definitely warranted! I think I will stick to using `create` for now and add further clarification.